crates/autopilot/src/domain/settlement/trade/math.rs

```diff
@@ -441,6 +410,27 @@ impl Trade {
     }
 }
 
+/// Fee per trade in a solution. These fees are taken for the execution of the
+/// trade.
+#[derive(Debug, Clone)]
+pub struct FeeBreakdown {
+    /// Fee reported by solvers, representing difference between UCP and custom
+    /// prices.
+    pub executed: eth::TokenAmount,
```

Comment:
Just by reading the struct and comments it's not really clear to me what this field is.
It sounds like this is just the sum of all protocol fees but I think since we need to account for the gas in the breakdown this is all protocol fees + gas fees.
In that case I think it would be nicer to have a field like `gas_costs` and a public member function `total_fees()`. WDYT?